<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_FluidSynthChannels</class>
 <widget class="QDialog" name="Dialog_FluidSynthChannels">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FluidSynth Channels</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(188, 188, 188);
border-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox_Dialog_Channels">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>330</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QFrame" name="frame_mainChannels">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>731</width>
     <height>311</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="midLineWidth">
    <number>1</number>
   </property>
   <widget class="QTabWidget" name="tabWidget_channels_table">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>711</width>
      <height>291</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(207, 207, 207);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_Channels_1_8">
     <attribute name="title">
      <string>Channels 1-8</string>
     </attribute>
     <widget class="QScrollArea" name="scrollArea_channels_1_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>691</width>
        <height>251</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>689</width>
         <height>249</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_Channels_9_16">
     <attribute name="title">
      <string>Channels 9-16</string>
     </attribute>
     <widget class="QScrollArea" name="scrollArea_channels_9_16">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>691</width>
        <height>251</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>689</width>
         <height>249</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_Fluid_Channels_SoundFonts">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>330</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>SoundFonts</string>
   </property>
  </widget>
  <zorder>frame_mainChannels</zorder>
  <zorder>buttonBox_Dialog_Channels</zorder>
  <zorder>pushButton_Fluid_Channels_SoundFonts</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox_Dialog_Channels</sender>
   <signal>accepted()</signal>
   <receiver>Dialog_FluidSynthChannels</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox_Dialog_Channels</sender>
   <signal>rejected()</signal>
   <receiver>Dialog_FluidSynthChannels</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
